scraper.py

```diff
@@ -0,0 +1,102 @@
+import json
+
+import requests
+import bs4
+from bs4 import BeautifulSoup, NavigableString, Tag
+
+
+# function to get URLs from page
+def get_url_title(url):
+    page = requests.get(url)
+    soup = BeautifulSoup(page.text, 'html.parser')
+    # finds all links to API/API Standards as listed on the page
+    links = [a['href'] for a in soup.find_all('a', href=True)]
+    filtereddivs = list(filter(lambda x: '/developer/api-catalogue/' in x, links))
+    beg = 'https://digital.nhs.uk'
+    urls = []
+    for link in filtereddivs:
+        url_combined = beg + link
+        urls.append(url_combined)
+    return urls
+
+
+# function to pull out the title of the API
+def get_api_title(api_url):
+    page = requests.get(api_url)
+    soup = BeautifulSoup(page.text, 'html.parser')
+    title = soup.find(class_='nhsd-t-heading-xxl nhsd-!t-margin-bottom-0').text
```

Comment:
we could make this a bit simpler by just doing `return soup.find(class_='nhsd-t-heading-xxl nhsd-!t-margin-bottom-0').text` rather than creating a new variable here